pkg/kotsadmconfig/validation/validator.go

```diff
@@ -0,0 +1,53 @@
+package validation
+
+import (
+	kotsv1beta1 "github.com/replicatedhq/kots/kotskinds/apis/kots/v1beta1"
+	configtypes "github.com/replicatedhq/kots/pkg/kotsadmconfig/types"
+)
+
+var (
+	validatableItemTypesMap = map[string]bool{
+		configtypes.EmptyItemType:    true,
+		configtypes.TextItemType:     true,
+		configtypes.PasswordItemType: true,
+		configtypes.TextAreaItemType: true,
+		configtypes.FileItemType:     true,
+	}
+)
+
+func isValidatableConfigItem(item kotsv1beta1.ConfigItem) bool {
+	if item.Validation == nil ||
+		item.Hidden ||
+		item.When == "false" ||
+		item.Repeatable ||
+		!validatableItemTypesMap[item.Type] {
+		return false
+	}
```

Comment:
I'm not sure if this is what was intended by @sgalsaleh's comment, but this would produce the desired behavior

Reply:
for the func isValidatableConfigItem
earlier I had this 
func isValidatableConfigItem(item kotsv1beta1.ConfigItem) bool {
	return item.Validation != nil && !item.Hidden && item.When != "false" && !item.Repeatable && validatableItemTypesMap[item.Type]
}
So, I switched it to if statement.
multiple conditional statements on if are allowed.
Switching to `or` instead of `and` is more readable, I think

Reply:
I meant separate if conditions. Separate if conditions is the Go convention and is much easier to read than this.

Reply:
ok